tests/ui/rfcs/rfc-2457-non-ascii-idents/utf8_idents.rs

```diff
@@ -1,14 +1,13 @@
+//! Check that non-ascii-idents are allowed.
+
 //@ check-pass
 //
 #![allow(mixed_script_confusables, non_camel_case_types)]
 
-fn foo<
-    'β,
-    γ
->() {}
```

Comment:
Question: I'm curious what happened to this test. The shape of this suggests to me that the test was previously exercising the diagnostics/suggestion spans of lints, maybe `mixed_script_confusables`. Does that lint have a dedicated suggestion / diagnostic span test?

I'll take a look later, unless you beat me to it.

Reply:
it used to be a feature gate test, see https://github.com/rust-lang/rust/pull/19073/files#diff-a1d248c23fe49ac33eeb944fb56d9f8bc4764565535c05efbbd7c3e83e487c5f

Reply:
Ah okay.